### DISCUSSION TITLE
Compiling motionplus with libcamera support on Raspbian bullseye

### DISCUSSION THREAD

**Discussion by grilparto** [Motion-Project/motionplus#35] [Q&A]
I'm trying to compile motionplus on raspbian bullseye.  When I do the ./configure I noticed in the output that it says "`libcamera support: no`.  I know that bullseye has libcamera support already included.  Unfortunately the package name isn't "libcamera" though I don't think.  It seems like it's either libcamera-apps or libcamera-apps-lite (for those using the "lite" version of the OS).

I have somewhat limited experience in this area, but is there a way to point the config file to the appropriate package when doing the ./configure?  I tried doing `./configure --with-libcam=/usr/lib/aarch64-linux-gnu`  (where the libcamera_app.so file is), but that didn't seem to work.  Looking at the CLI output higher up it looks like it appended `/lib/pkgconfig` to the directory I included in the option flag.  

Specifically, this was the output when I used the `--with-libcam=/usr/lib/aarch64-linux-gnu` flag:
`checking LIBCAM pkgconf path... /usr/lib/aarch64-linux-gnu/lib/pkgconfig:`
`checking for LIBCAM... no`

Anyone have any thoughts?  I'm 98% sure I'm just on the wrong track here because of my limited expertise.

**Comment by Mr-Dave** [comment]
You must install the libcamera apps via apt to get that support.

**Comment by martin762** [comment]
I think that you will need libcamera-dev via apt

**Reply by Bhoft** [reply]
oh man why is this nowhere written or mentioned in the docs / issues on github that you also have to install libcamera-dev to that configure finds the libaries...
this had taken me ages to find.

can somebody also give examples what prefix / sysconfdir must be set so thet the files get into the correct directory?
